src/Tribe/Tickets.php

```diff
@@ -402,7 +402,9 @@ public function get_ticket_delete_link( $ticket = null ) {
 				return;
 			}
 
-			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) ? __( 'Delete RSVP', 'event-tickets' ) : __( 'Delete Ticket', 'event-tickets' ) ;
+			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class )
```

Comment:
```suggestion
			$delete_text = _x( 'Delete %s', 'delete link', 'event-tickets' );

			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class )
```

Reply:
I thought about doing it this way, too. 👍